Traducir.Web/src/App/Components/Users.tsx

```diff
@@ -40,40 +40,13 @@ export default class Users extends React.Component<IUsersProps, IUsersState> {
                     </tr>
                 </thead>
                 <tbody>
-                    {this.state.users.map(u =>
-                        <tr key={u.id}>
-                            <td>
-                                <a href={`https://${this.props.config.siteDomain}/users/${u.id}`} target="_blank">
-                                    {u.displayName} {u.isModerator ? "♦" : ""}
-                                </a>
-                            </td>
-                            <td>{userTypeToString(u.userType)}</td>
-                            <td>{this.props.currentUser && this.props.currentUser.canManageUsers &&
-                                <div className="btn-group" role="group">
-                                    <Link to={`/suggestions/${u.id}`} className="btn btn-sm btn-primary">view suggestions</Link>
-                                    {u.userType === UserType.User &&
-                                        <button type="button" className="btn btn-sm btn-warning" onClick={e => this.updateUserType(u, UserType.TrustedUser)}>
-                                            Make trusted user
-                                        </button>
-                                    }
-                                    {u.userType === UserType.Banned &&
-                                        <button type="button" className="btn btn-sm btn-warning" onClick={e => this.updateUserType(u, UserType.User)}>
-                                            Lift Ban
-                                        </button>
-                                    }
-                                    {u.userType === UserType.TrustedUser &&
-                                        <button type="button" className="btn btn-sm btn-danger" onClick={e => this.updateUserType(u, UserType.User)}>
-                                            Make regular user
-                                        </button>
-                                    }
-                                    {u.userType !== UserType.Banned && u.userType !== UserType.TrustedUser && u.userType !== UserType.Reviewer && !u.isModerator &&
-                                        <button type="button" className="btn btn-sm btn-danger" onClick={e => this.updateUserType(u, UserType.Banned)}>
-                                            Ban User
-                                        </button>
-                                    }
-                                </div>
-                            }</td>
-                        </tr>)}
+                    {this.state.users.map(u => <User
+                        key={u.id}
+                        user={u}
+                        config={this.props.config}
+                        refreshUsers={this.refreshUsers}
+                        showErrorMessage={this.props.showErrorMessage}
```

Comment:
you're missing `currentUser` ;) that's breaking the "view suggestions" button for regular users and all the reviewers actions.